compiler/src/iree/compiler/Codegen/Sandbox/Passes.td

```diff
@@ -67,6 +67,22 @@ def LinalgFuse : Pass<"linalg-fuse", "func::FuncOp"> {
   ];
 }
 
+def LinalgSplitReduction : Pass<"linalg-split-reduction", "func::FuncOp"> {
+  let summary = "Pass to splitReduce linalg operations.";
+  let constructor = "mlir::createLinalgSplitReductionPass()";
+  let dependentDialects = [
+    "::mlir::linalg::LinalgDialect"
+  ];
+  let options = [
+    // This corresponds to the value in the last level of tiling
+    Option<"size", "size", "int64_t", /*default=*/"0",
+      "Size that splitReduction should use for parallel reduction results.">,
```

Comment:
IMO, we can name it `ratio`. That's the naming from upstream.

Reply:
Ratio seems a completely random name, even if upstream uses it. What I need is the final size of the parallel dimension. I would strongly favor leaving it as is.

Reply:
Size=0 triggers reading the lowering_config and if there's no lowering_config, the pass doesn't execute. Executing the pass with tile size of 1 is strictly worse than not executing it (and instead using the default linalg_tile_and_fuse)

Reply:
Thanks for the explanation, please document it in the td file as well, thanks!